go/private/library.bzl

```diff
@@ -16,7 +16,7 @@ load("@io_bazel_rules_go//go/private:common.bzl", "get_go_toolchain", "DEFAULT_L
 load("@io_bazel_rules_go//go/private:asm.bzl", "emit_go_asm_action")
 load("@io_bazel_rules_go//go/private:providers.bzl", "GoLibrary", "CgoLibrary")
 
-def emit_library_actions(ctx, srcs, deps, cgo_object, library, want_coverage):
+def emit_library_actions(ctx, srcs, deps, cgo_object, library, want_coverage, importpath, golibs=[]):
```

Comment:
it has to be importpath, it's stored on the returned provider as that.
It really must be the import path for the generated library, it's just that we generate synthetic non importable packages for main/test packages, and I can't just call them by their name because they can collide. I could pass a name separately from an import path, and then verify that they were the same if importpath is set, but that seems fairly pointless?